src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,695 @@
+				```
+			*/
+			dropDownAttachedToBody: false,
+			/* type="number" Gets/Sets show/hide drop-down list animation duration in milliseconds.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+				  dropDownAnimationDuration: 500
+				});
+
+				//Get
+				var animationShowDuration= $(".selector").%%WidgetName%%("option", "dropDownAnimationDuration");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "dropDownAnimationDuration", 500);
+			```
+			*/
+			dropDownAnimationDuration: 300,
+			/* type="string" Gets/Sets the maximum value for the dropdown items. The format should be the same as the inputTimeFormat
+				Note! This option can not be set runtime.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+				  maxTimeValue: "18:00"
+				});
+
+				//Get
+				var maxTimeValue= $(".selector").%%WidgetName%%("option", "maxTimeValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "maxTimeValue", "18:00");
+			```
+			*/
+			maxTimeValue: null,
+			/* type="string" Gets/Sets the minimum value for the dropdown items. The format should be the same as the inputTimeFormat
+				Note! This option can not be set runtime.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+				  minTimeValue: "09:00"
+				});
+
+				//Get
+				var minTimeValue= $(".selector").%%WidgetName%%("option", "minTimeValue");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "minTimeValue", "09:00");
+			```
+			*/
+			minTimeValue: null,
+			/* type="dropdown|clear|spin" Gets visibility of the spin, clear and drop-down button. That option can be set only on initialization. Combinations like 'dropdown,spin' or 'spin,clear' are supported too.
+				Note! This option can not be set runtime.
+				```
+					//Initialize
+					$(".selector").%%WidgetName%%({
+						buttonType : "dropdown"
+					});
+
+					//Get
+					var button = $(".selector").%%WidgetName%%("option", "buttonType");
+
+					//Initialize multiple buttons
+					$(".selector").%%WidgetName%%({
+						buttonType : "dropdown,clear"
+					});
+				```
+				dropdown type="string" A button to open/close the list is located on the right side of the editor.
+				clear type="string" A button to clear the value is located on the right side of the editor.
+				spin type="string" Spin buttons are located on the right side of the editor.
+			*/
+			buttonType: "dropdown",
+			/* @Ignored@ Removed from timepicker options*/
+			minValue: null,
+			/* @Ignored@ Removed from timepicker options*/
+			maxValue: null,
+			/* @Ignored@ Removed from timepicker options*/
+			dateDisplayFormat: null,
+			/* @Ignored@ Removed from timepicker options*/
+			dateInputFormat: null,
+			/* @Ignored@ Removed from timepicker options*/
+			yearShift: null,
+			/* @Ignored@ Removed from timepicker options*/
+			displayTimeOffset: null,
+			/* @Ignored@ Removed from timepicker options*/
+			enableUTCDates: null

```

Comment:
This might be okay to leave in, actually. All this option does is to change the default value stored in the submit input, where:
"2018-01-26T13:32:42.540+02:00" would be submitted as "2018-01-26T11:32:42.540Z" instead.
It's really a matter of preference for the dev, depending on the backend, though in most cases it makes no difference.